d3ledger/back-office-admin
# D3 1058 | Merge admin functions from client to admin

# Description
https://soramitsu.atlassian.net/browse/D3-1058

# Works done
- [x] move fee settings
- [x] move transaction explorer
- [x] move dependent small functions

# How to test
0. Run back-office, back-office-admin, notary, report-system
1. Make some transactions
2. Open back-office-admin as admin
3. Check transaction explorer and fee settings
